.gitignore

```diff
@@ -15,3 +15,6 @@ buildNumber.properties
 /give-me-a-name.iml
 /.idea/
 .vscode
+
+# Compiled class files
+*.class
```

Comment:
Why do you even generate class files in source folders? You should be building with Maven, which already has its `target` folders in `.gitignore`. Please delete all class files that you committed and remove this unnecessary entry.